src/site/content/en/react/how-to-use-amp-in-nextjs-and-why/index.md

```diff
@@ -0,0 +1,272 @@
+   import React from 'react';
+   import { useAmp } from 'next/amp';
+
+   export const config = { amp: 'hybrid' };
+
+   const Home = () => (
+     useAmp() ? (
+       <p>This is the <strong>AMP</strong> version of the home page</p>
+     ) : (
+       <p>This is the main version of the home page</p>
+     )
+   );
+
+   export default Home;
+   ```
+
+1. Load the main version of the page:
+
+   <figure>
+     <img src="main.png"
+          alt="The main version of the page.">
+     <figcaption>
+       The main version of the page.
+     </figcaption>
+   </figure>
+
+1. Add `?amp=1` to the end of the URL again to load the AMP version of the page:
+
+   <figure>
+     <img src="amp.png"
+          alt="The AMP version of the page.">
+     <figcaption>
+       The AMP version of the page.
+     </figcaption>
+   </figure>
+
+1. Try rendering AMP's replacement of the image tag, `amp-img`:
+
+   {# TODO(kaycebasques): Line highlighting isn't working with this sample. #}
+
+   ```jsx
+   import React from 'react';
+   import { useAmp } from 'next/amp';
+
+   export const config = { amp: 'hybrid' };
+
+   const imgSrc = 'https://placekitten.com/1000/1000';
+
+   const Image = () => (
+     useAmp() ? (
+       <amp-img alt="A cute kitten"
+         src={imgSrc}
+         width="1000"
+         height="1000"
+         layout="responsive">
+       </amp-img>
+     ) : (
+       <img alt="A cute kitten"
+         src={imgSrc}
+         width="1000"
+         height="1000">
+       </img>
+     )
+   );
+
+   const Home = () => (
+     <div>
+       <Image />
+     </div>
+   );
+
+   export default Home;
+   ```
+
+   `layout="responsive"` automatically renders a fully responsive image with an aspect ratio
+   specified by width and height. Check out [Layout & media queries][layout] to learn more about
+   the supported layouts of AMP elements, and [amp-img] to learn more about that element's
+   optimizations.
+    
+1. View the main version of the page again.
+
+   <figure>
+     <img src="mainimg.png"
+          alt="The main version of the image.">
+     <figcaption>
+       The main version of the image.
+     </figcaption>
+   </figure>
+
+1. View the AMP version of the page again.
+
+   <figure>
+     <img src="ampimg.png"
+          alt="The AMP version of the image.">
+     <figcaption>
+       The AMP version of the image.
+     </figcaption>
+   </figure>
+
+### How to create AMP-only pages {: #amponly }
+
+Next.js also supports AMP-only pages. With this approach, a single AMP page is served and rendered
+to users and search engines at all times.
+
+1. To render an AMP-only page, change the value of the `amp` property in the config object to `true`.
+
+   ```jsx
+   import React from 'react'
+
+   export const config = { amp: true };
+
+   const Home = () => (
+     <p>This is an AMP-only page</p>
+   );
+     
+   export default Home;
+   ```
+
+## How to decide whether to use the Hybrid AMP or AMP-only approach {: #guidance }
```

Comment:
Helping developers decide which framework/toolchain/etc. to use will be a big theme in web.dev content over 2020. We might want to move this to a separate page one day. But for now I think it works well as a section in this guide.